sys/linux/dev_video4linux_vim2m_amd64.const

```diff
@@ -0,0 +1,22 @@
+# AUTOGENERATED FILE
+AT_FDCWD = 18446744073709551516
```

Comment:
For any new files it would be good to get const files for all archs. Please try syz-env image.